src/AbstractGithubObject.php

```diff
@@ -9,7 +9,7 @@
 namespace Joomla\Github;
 
 use Joomla\Http\Exception\UnexpectedResponseException;
-use Joomla\Http\Http as BaseHttp;
+use Joomla\Http\Http;
```

Comment:
Please revert this change - there is already a class in this package called Http https://github.com/joomla-framework/github-api/blob/f58a08de8c8238712a67406356e78c8ac13ceb74/src/Http.php

Reply:
Fixed, thank you!